ssawg_trending_scraper.py

```diff
@@ -41,11 +43,10 @@
 
 
 def get_opt():
-    parser = argparse.ArgumentParser(description='Make SSAWG trending page')
-    parser.add_argument('--data-dir',
-                        type=str,
-                        default='.',
-                        help='Output data directory')
+    parser = argparse.ArgumentParser(description="Make SSAWG trending page")
+    parser.add_argument(
+        "--data-dir", type=str, default=".", help="Output data directory"
+    )
 
     args = parser.parse_args()
```

Comment:
Should this return the parser according to best practices?

Reply:
Probably, but for this particular application the old style seems fine.